code/src/index.css

```diff
@@ -1,13 +1,151 @@
 body {
-  margin: 0;
+  margin: 10px;
   font-family: -apple-system, BlinkMacSystemFont, "Segoe UI", "Roboto", "Oxygen",
     "Ubuntu", "Cantarell", "Fira Sans", "Droid Sans", "Helvetica Neue",
     sans-serif;
   -webkit-font-smoothing: antialiased;
   -moz-osx-font-smoothing: grayscale;
+  background-color: rgb(26, 24, 24);
 }
 
 code {
   font-family: source-code-pro, Menlo, Monaco, Consolas, "Courier New",
     monospace;
 }
+
+a {
+  text-decoration: none;
+}
+
+.title {
+  font-size: 20px;
+  font-family: Helvetica Neue;
+  color: white;
+  font-size: 25px;
+  font-weight: 500;
+  text-align: center;
+}
+
+.all-albums {
+  display: grid;
+  justify-content: center;
+  align-items: center;
```

Comment:
the layout looks super great in the set breakpoint but it gets a little crammed in some transition widths. One column gets cut off right before the switch to tablet size from desktop